### PR TITLE
Replaced hardcoded HTML ids in Carnation theme to not break inheritance of CSS styles and JS logic in sub-themes

### PR DESCRIPTION
**Pull request by afi13**
Carnation's sub-themes don't look OK because of using IDs elements which contain the name of the current theme. Such IDs have been replaced with new classes, CSS and JS have been updated to follow new structure.
…

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature, please create PR against 8.x-1.xx-dev branch.

If this is a bug fix - use 8.x-1.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [x] Enable carnation theme.
- [x] Make sure footer menu is displayed correctly.
- [x] Make sure local tabs are displayed correctly.
- [x] Make sure orange banner on 'Locations' page is displayed correctly.
- [x] There no any visible regression related to grid or styles.

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [x] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_carnation/src/scss/modules/_footer.scss [hunk 1]
```diff
@@ -124,7 +124,7 @@
     font-size: 15px;
     color: $gray-400;
 
-    #block-openy-carnation-footermenuright {
+    .block-va-ymca-footer-menu-right-menu {
```
Comment (ddrozdik) [inline themes/openy_themes/openy_carnation/src/scss/modules/_footer.scss]: Please use not related to the specific project namespace. I recommend something like this
```suggestion
    .block-openy-carnation-footer-menu-right-menu {
```

File: themes/openy_themes/openy_carnation/templates/block/block--system-menu-block--footer-menu-right.html.twig [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * technology. Headings allow screen-reader and keyboard only users to navigate
+ * to or skip the links.
+ * See http://juicystudio.com/article/screen-readers-display-none.php and
+ * http://www.w3.org/TR/WCAG-TECHS/H42.html for more information.
+ *
+ * @ingroup themeable
+ */
+#}
+{% set heading_id = attributes.id ~ '-menu'|clean_id %}
+<nav role="navigation" aria-labelledby="{{ heading_id }}"{{ attributes.addClass('block-va-ymca-footer-menu-right-menu') |without('role', 'aria-labelledby') }}>
```
Comment (ddrozdik) [inline themes/openy_themes/openy_carnation/templates/block/block--system-menu-block--footer-menu-right.html.twig]: change class, as mentioned above.

**Review by ddrozdik — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_carnation/src/scss/modules/_footer.scss (hunk 1 above)
Comment (afi13) [reply themes/openy_themes/openy_carnation/src/scss/modules/_footer.scss]: Fixed

File: themes/openy_themes/openy_carnation/templates/block/block--system-menu-block--footer-menu-right.html.twig (hunk 2 above)
Comment (afi13) [reply themes/openy_themes/openy_carnation/templates/block/block--system-menu-block--footer-menu-right.html.twig]: Fixed

**Review by ddrozdik — APPROVED** [review]

**Comment by ddrozdik** [comment]
ok to test

**Review by podarok — APPROVED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1154 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1154 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1154 |
| Installation process | http://install.openy.cibox.tools/build1154/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1154/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1154/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1154/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1154/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1154/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1154/blogWCAG2AAhtmlcs.txt

**Comment by ddrozdik** [comment]
retest this please

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1157/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1157/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1157/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1157/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1157/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1157 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1157 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1157 |
| Installation process | http://install.openy.cibox.tools/build1157/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1157/

**Pull request merged**